community-operators/planetscale/planetscale-operator.v0.1.8.clusterserviceversion.yaml

```diff
@@ -0,0 +1,292 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: planetscale-operator.v0.1.8
+  namespace: placeholder
+  annotations:
+    alm-examples: |-
```

Comment:
This CSV is missing the following annotation:
`metadata.annotations.capabilities` should be added with the support-level for the operator. The 5 levels can be found in this diagram: https://www.operatorhub.io/operator/images/capability-level-diagram.svg
These fields are requested for displaying this operator properly in the OperatorHub UI.

Reply:
Added "Deep Insights", will be in the next push